dart/lib/src/protocol/sentry_log.dart

```diff
@@ -0,0 +1,13 @@
+import 'sentry_log_item.dart';
+
+class SentryLog {
+  final List<SentryLogItem> items;
+
+  SentryLog({required this.items});
+
+  Map<String, dynamic> toJson() {
+    return {
+      'items': items.map((e) => e.toJson()).toList(),
+    };
+  }
+}
```

Comment:
is there a spec that we need a 'parent' class that holds multiple logs?

imo we can just define each item as a log

Reply:
It's just the way that the envelope structure looks, but I agree, we don't really need it at the model level and we can just pass a lost of log (items) to the envelope factory methods.